pkg/orchestration/wiring/sqs/plugin.go

```diff
@@ -79,18 +79,31 @@ func WireUp(stateResource *orch_v1.StateResource, context *wiringplugin.WiringCo
 	}
 	wiredResources = append(wiredResources, serviceInstance)
 
+	var hasDeadLetterQueue bool
+	if stateResource.Spec != nil {
+		var spec map[string]interface{}
```

Comment:
I think it'd be better to unmarshal into an anonymous struct with just one field to avoid storing all the results that are not need.